excel/beginner/basics-of-excel/what-is-a-workbook.md

```diff
@@ -0,0 +1,41 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+
+aspects:
+  - new
+  - workout
+
+type: normal
+
+category: how to
+
+---
+
+# What is a workbook?
+
+---
+## Content
+
+A workbook is a single file containing one or more worksheets(sheets).
+
+When you create a new workboot, by default, you get 1 worksheet. More sheets can be added at any time.
+
+The main difference between a workbook and a worksheet(sheet) is that you can have many worksheets inside a single workbook, but not vice versa. The workbook is the main file and it can contain as many worksheets as needed.
+
+A Worksheet is a grid-like table sheet compromised of rows and columns.
+
+To create a new workbook in Microsoft Excel you need to go to:
+ `File > New > *blank workbook*`
+
+The Google docs version is almost the same except instead of a "blank workbook" it's called "spreadsheet".

```

Comment:
```suggestion
The Google sheets version is almost the same except instead of a "blank workbook" it's called "spreadsheet".

```

Reply:
Should I change this to Sheets or leave it as Docs?

In my opinion: 
The Google Docs version of Excel is Spreadsheet.

Am confused So I will wait for your responce